runbooks/how-to-rerun-failed-pipeline-load.md

```diff
@@ -0,0 +1,61 @@
+# How to Re-run a Failed BFD Pipeline Load
+
+Follow this runbook to successfully reload data via BFD pipeline after a failed run.
+
+Note: If there are recently merged code changes or pending redeploys, make sure those related deployments finish before running these steps. 
+
+1. SSH into the AWS ETL EC2 instance for a given environment ```bfd-<test/prod/prod-sbx>-etl``` by running ```ssh -i <local ssh key> <aws username>@<EC2 IP Address>```. The entry point is the specific AWS user home directory.
+
+2. Confirm the pipeline has failed to load data
+    - Inside the EC2 Instance run ```sudo systemctl status bfd-pipeline```. The output should NOT say "active(running) since …".
```

Comment:
Not 100% sure but does every failed load blow up the pipeline, such that we'd always expect it to be not running if it fails a load? Just checking if this would always be the case.

Reply:
Good catch